bug_change_status_page.php

```diff
@@ -153,6 +153,9 @@
 			<?php
 				if( $f_new_status >= $t_resolved ) {
 					if( relationship_can_resolve_bug( $f_bug_id ) == false ) {
+						if( OFF == config_get( 'allow_parent_of_unresolved_to_close' ) ) {
```

Comment:
Does this error apply for both web UI and APIs?

Reply:
Just web UI.

Before this change, we had it just on bug update page.
The idea is to throw the error as early as possible to avoid unneeded entering of data.
e.g. before the change the user typically set `Resolution`, set `Fixed in Version` and added a note and got the error message after clicking the `Resolve Issue` button.

We might want to have this check also in function validate of BugData class, but that's another story.